examples/aggregator/handler_module/src/lib.rs

```diff
@@ -14,6 +14,30 @@
 // limitations under the License.
 //
 
+//! Aggregator example for Project Oak.
```

Comment:
Is not it better to have this documentation in the `main_module`?

Reply:
Done